src/Sulu/Bundle/SecurityBundle/Resources/config/services.xml

```diff
@@ -54,7 +54,7 @@
 
         <service id="sulu_security.authentication_handler" class="%sulu_security.authentication_handler.class%">
             <argument type="service" id="router"/>
-            <argument type="service" id="session"/>
+            <argument type="service" id="session" on-invalid="null"/>
```

Comment:
That's actually not really nice, but I can't find a better way of doing this :confused: